### PR TITLE
[audit] fix(inline-loader): updates to loader svg

### LINKED ISSUE #1906: fix[Loading Component]: both loading and inline loading component tweaks
## fix[Loading Component]: small loading and inline loading

- [x] Get rid of clunky toggle state button and try a timer Loading...loading... success 

- [x] Convert button to `sm` variant

- [x] Check small loader (both loading and inline loading) stoke weight looks narrower than spec (it should be 3px wide in spec and 16px square in total)
image

- [x] Small loader is missing the gray section at the start, then gray section suddenly appears.
gif

- [x] Don't we need a failure state beside success state? See failure state design below:
image

### PR DESCRIPTION
**Pull request by dakahn**
Closes #1906 

Updates to the loader SVG a la our component audit. 

#### Changelog

**Changed**

- widened circle stroke-width of circles to 3px
- adjusted dash offset so it matches new overall svg width
- adjusts stroke-width of checkmark 1.8px
- adjusts overall svg width to 16px

### CODE REVIEW

File: src/components/inline-loading/inline-loading.hbs [hunk 1]
```diff
@@ -11,9 +11,9 @@
     <div data-inline-loading-spinner class="{{@root.prefix}}--loading {{@root.prefix}}--loading--small">
       <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
         {{#if componentsX}}
-        <circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />
+        <circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="30.5" />
         {{/if}}
-        <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="37.5" />
+        <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="30" />
```
Comment (asudoh) [inline src/components/inline-loading/inline-loading.hbs]: Seems that the `r` attribute here is applied to `v9` as well, and thus the change here need to be roped off.

**Review by asudoh — COMMENTED** [review]
LGTM basically, just one thing - Thanks @dakahn!

**Review by shixiedesign — COMMENTED** [review]
Awesome. Thanks for making the change! I noticed two things:

- [x] Inline loader is looking good on Chrome but **on Safari**, the loader wobbles a bit. Seems like the center of rotation is mis-aligned with the image. 
inline-loader

- [x] [Small loader](https://deploy-preview-2004--the-carbon-components.netlify.com/?nav=loading) should have the same styling as the inline loader. A 3px stroke weight. 
image

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @dakahn!

**Review by shixiedesign — APPROVED** [review]
Awesome! 🎉

**Pull request merged**